### ISSUE TITLE
GLib.Error from manual install

### ISSUE THREAD

**Issue by SamuelWN** [calexandru2018/linux-gui-legacy#85]
**Describe the bug**
After encountering the [make_ovpn_template](https://github.com/ProtonVPN/linux-gui/issues/83) issue installing from the pip3 repo, I tried manual install method, but encountered the following error on launch:

    Traceback (most recent call last):
      File "/usr/local/bin/protonvpn-gui", line 11, in <module>
        load_entry_point('protonvpn-gui', 'console_scripts', 'protonvpn-gui')()
      File "/home/samuelwn/Downloads/linux-gui/linux_gui/protonvpn_gui.py", line 66, in init
        style_provider.load_from_path(UI_STYLES)
    GLib.Error: gtk-css-provider-error-quark: common.css:51:20'-gtk-icon-effect' is not a valid property name (3)

**To Reproduce**

    sudo apt install -y python3-gi python3-gi-cairo gir1.2-gtk-3.0 gir1.2-appindicator3 libnotify-bin
    git clone https://github.com/ProtonVPN/linux-gui.git
    cd linux-gui
    sudo pip3 install -e .
    sudo protonvpn-gui

**Expected behavior**
ProtonVPN GUI to launch

**Desktop:**
 - OS: Ubuntu 16.04

**Additional context**
The manually-installed version gives an error during uninstallation that I did not encounter when installing from the pip repository, though I think it might make sense to make it a separate report.

    $ sudo pip3 --verbose uninstall protonvpn-gui 
    WARNING: The directory '/home/samuelwn/.cache/pip' or its parent 
        directory is not owned or is not writable by the current user. 
        The cache has been disabled.
        Check the permissions and owner of that directory.
        If executing pip with sudo, you may want sudo's -H flag.
    Found existing installation: protonvpn-gui 2.1.0
    Not sure how to uninstall: protonvpn-gui 2.1.0 - Check: /home/samuelwn/Downloads/linux-gui
    Can't uninstall 'protonvpn-gui'. No files were found to uninstall.

    $ sudo pip3 show protonvpn-gui
    WARNING: The directory '/home/samuelwn/.cache/pip' or its parent 
        directory is not owned or is not writable by the current user. 
        The cache has been disabled.
        Check the permissions and owner of that directory.
        If executing pip with sudo, you may want sudo's -H flag.
    Name: protonvpn-gui
    Version: 2.1.0
    Summary: Linux GUI client for ProtonVPN
    Home-page: https://github.com/ProtonVPN/linux-gui
    Author: Proton Technologies AG
    Author-email: contact@protonvpn.com
    License: GPLv3
    Location: /home/samuelwn/Downloads/linux-gui
    Requires: protonvpn-cli, requests
    Required-by: 

(executing with `sudo -H` does  not resolve the issue and produces the same error)

**Comment by SamuelWN** [comment]
After manually uninstalling proton-gui and reverting my protonvpn-cli instance to version 2.2.2, I got a new, different, GLib error which may point to the issue:

    Traceback (most recent call last):
      File "/usr/local/bin/protonvpn-gui", line 8, in <module>
        sys.exit(initialize_gui())
      File "/usr/local/lib/python3.5/dist-packages/protonvpn_linux_gui/gui.py", line 738, in initialize_gui
        interface.add_from_file(glade_path[:-1])
    GLib.Error: gtk-builder-error-quark: /usr/local/lib/python3.5/dist-packages/protonvpn_linux_gui/resources/main.glade:26:1 Required gtk+ version 3.20, current version is 3.18 (7)

> Required gtk+ version 3.20, current version is 3.18

**Comment by calexandru2018** [comment]
`sudo protonvpn-gui` this is your problem if you install it via Github. Please follow the instructions in the readme file.

To install: `sudo pip3 install -e .`

To launch: `protonvpn-gui`

Once you get this solved, please close the issue :)

**Comment by SamuelWN** [comment]
Unfortunately, running as a standard user produces the same result.

    $ protonvpn-gui 
    Traceback (most recent call last):
      File "/usr/local/bin/protonvpn-gui", line 11, in <module>
        load_entry_point('protonvpn-gui', 'console_scripts', 'protonvpn-gui')()
      File "/home/samuelwn/Downloads/linux-gui/linux_gui/protonvpn_gui.py", line 66, in init
        style_provider.load_from_path(UI_STYLES)
    GLib.Error: gtk-css-provider-error-quark: common.css:51:20'-gtk-icon-effect' is not a valid property name (3)

**Comment by calexandru2018** [comment]
Thanks for the info @SamuelWN 

Based on you previous post, it is not the exact error. This time a property is being flagged as not valid, which is weird since by GTK+ docs, this property [ should be supported  by GTK3  v3.18](https://developer.gnome.org/gtk3/stable/chap-css-properties.html). I will try to recreate the issue with Ubuntu 16.04.

**Comment by SamuelWN** [comment]
No prob. 

And sorry, my last comment may've been a bit confusing. 
What I meant was that running the GitHub version of the GUI (with `protonvpn-cli` v2.2.4) produced the same error message with / without sudo (the error message matches the one I quoted in the "Describe the bug" section of the original post). 

The error in my [earlier comment](https://github.com/ProtonVPN/linux-gui/issues/85#issuecomment-645374522) was the produced by reverting `protonvpn-cli` to v2.2.2 and attempting to use the pip3 repo version of the gui.

Sorry for any confusion.

**Comment by calexandru2018** [comment]
Hi again @SamuelWN 

I tried to replicate the issue and I can confirm it. Now the thing is, the GUI is based on GTK+3.22, and this version of GTK is only supported from Ubuntu 18.04 bionic and higher. I tried to search and see if the newer GTK version could be installed in Ubuntu 16.04, but only GTK+3.20 is supported, **meaning that if you want to use the GUI you will need to upgrade to Ubuntu 18.04 bionic.**

As it is at the moment, there are no plans to support older versions of Ubuntu, although that might possibly change in the future.
